mopidy_mpd/protocol/tagtype_list.py

```diff
@@ -3,19 +3,18 @@
     "ArtistSort",
     "Album",
     "AlbumArtist",
-    "AlbumArtistSort",
```

Comment:
Don't think should remove this.

Reply:
It didn't appear this was actually being used anywhere, but I added it back in.